pallets/spaces/src/benchmarking.rs

```diff
@@ -0,0 +1,87 @@
+//! Spaces pallet benchmarking.
+
+use super::*;
+
+use frame_benchmarking::benchmarks;
+use frame_benchmarking::whitelisted_caller;
+use frame_support::ensure;
+use frame_support::pallet_prelude::Get;
+use frame_system::RawOrigin;
+use sp_std::vec;
+
+use crate::Config;
+use crate::types::*;
+
+use pallet_permissions::{
+    default_permissions::DefaultSpacePermissions,
+    SpacePermissionSet,
+};
+
+
+fn dummy_space_content() -> Content {
+    Content::IPFS(b"QmRAQB6YaCaidP37UdDnjFY5aQuiBrbqdyoW1CaDgwxkD4".to_vec())
+}
+
+fn get_next_space_id<T: Config>() -> SpaceId {
+    NextSpaceId::<T>::get()
+}
+
+fn get_new_space_id<T: Config>() -> SpaceId {
+    let space_id = get_next_space_id::<T>();
+    NextSpaceId::<T>::mutate(|n| *n += 1);
+    space_id
+}
+
+fn create_dummy_space<T: Config>() -> Space<T> {
+    let id = get_new_space_id::<T>();
+    let space = Space::new(
+        id,
+        None,
+        whitelisted_caller::<T::AccountId>(),
+        Content::None,
+        None,
+    );
+    SpaceById::<T>::insert(id, space);
+
+    SpaceById::<T>::get(id).expect("Expected space to exist")
+}
+
+benchmarks! {
+    create_space {
+        let caller = whitelisted_caller::<T::AccountId>();
+
+        let parent_space = create_dummy_space::<T>();
+        let new_space_id = get_next_space_id::<T>();
+
+        let parent_id_opt = Some(parent_space.id);
+        let content = Content::None;
+        let permissions_opt = None;
```

Comment:
Same as with content. The worst case here if we put all the permissions in `SpacePermissions.none` set.
For now let's leave it None, but we have to get rid of this argument in `create_space`, and don't allow to override space permissions when creating.